website/content/docs/job-specification/action.mdx

```diff
@@ -0,0 +1,127 @@
+---
+layout: docs
+page_title: action Block - Job Specification
+description: The "action" block allows for executable commands that operators
+
+---
+
+# `action` Block
+
+<Placement groups={['job', 'group', 'task', 'action']} />
+
+The `action` block allows job authors to define custom commands. These commands
+can be executed by operators with the necessary permissions on a running
+allocation, offering a controlled way to interact with tasks.
+
+## `action` Parameters
+
+The `action` block is used within a task and has the following parameters:
+
+- `command` (Required): Specifies the command to be executed.
+- `args` (Optional): Provides a list of arguments to pass to the command.
```

Comment:
```suggestion
- `command` `(string: <required>)` - Specifies the command to be executed.
- `args` `(array<string>: [])`: Provides a list of arguments to pass to the command.
```

We have a weird mishmash of how these are defined, but this is the pattern we use for job spec.